portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,112 @@
 
 "use strict"
 
+// is called when page loads
+document.addEventListener("DOMContentLoaded", () => {
+    // set max-comments variable
+    const DEFAULT_COMMENTS_NUMBER = 3;
+    const MAX_COMMENTS_NUMBER = 10;
+    let maxComments = DEFAULT_COMMENTS_NUMBER;
+    // set event to change maxComments variable
+    const maxCommentsElement = document.getElementById("max-comments");
+    maxCommentsElement.addEventListener("change", (event) => {
+        const inputMaxComments = parseInt(event.target.value);
+        // validation of maxComments number
+        if (isNaN(inputMaxComments)) {
+            displayErrorInput(maxCommentsElement, "Please, write a number in Arabic numerals");
+            return;
+        }
+        if (inputMaxComments < 0) {
+            displayErrorInput(maxCommentsElement, "Please, write a positive number");
+            return;
+        }
+        if (inputMaxComments > MAX_COMMENTS_NUMBER) {
+            displayErrorInput(maxCommentsElement, `Please, write a number less than ${MAX_COMMENTS_NUMBER}`);
+            return;
+        }
+        maxComments = inputMaxComments;
+        // reload comments if valid
+        loadComments(maxComments);
+    });
+    // set event to delete comments
+    const buttonDeleteComments = document.getElementById("comments-delete");
+    buttonDeleteComments.addEventListener("click", deleteAllComments);
+    // add validation to comment-add form
+    const commentAddForm = document.getElementById("comment-add-form");
+    commentAddForm.addEventListener("submit", commentAddFormValidate);
+    // initially load comments
+    loadComments(maxComments);
+});
+
+function commentAddFormValidate(event) {
```

Comment:
Could you add a comment to brief what this function does?

Same for the other functions.